app/models/case/sar/offender.rb

```diff
@@ -505,6 +505,10 @@ def set_valid_case_number
     self.number = next_number
   end
 
+  def is_high_profile?
```

Comment:
Is this method needed?

Reply:
If it were needed, `flag_as_high_profile?` is a boolean, so it will already return true or false. You don't need to check it's value.